### ISSUE TITLE
[Mozilla Firefox] Page crashes on load

### ISSUE THREAD

**Issue by ThePerkinrex** [HermesMessenger/Hermes#86]
There's some error with the css and adding rules to the stylesheets
`TypeError: css[(css.length - 1)].addRule is not a function utils.js:144:33`

**Comment by ThePerkinrex** [comment]
We could use the `insertRule` instead of `addRule`
For refereence the code is:
```js
134        let cssRuleExists = false;
135        let css = document.styleSheets;
136
137        for (var r = 0; r < css[css.length - 1].rules; r++) {
138            if (css[css.length - 1].rules[r].selectorText.includes(`user-${quoted_user_id}`)) {
139                cssRuleExists = true;
140                break
141            }
142        }
143        if (!cssRuleExists) {
144            css[css.length - 1].addRule(`.quote.user-${quoted_user}`, quote_css);
145        }
```

**Comment by spaceface777** [comment]
The issue is caused by the quotes.

**Comment by spaceface777** [comment]
Fixed in 13f19ea
